src/components/Section/Section.css

```diff
@@ -11,6 +11,10 @@
   padding-bottom: 100px;
 }
 
+.section_type_products {
+  padding-bottom: 270px;
```

Comment:
Changing this to `min-height: 324px` will give a height of 544px, matching the figma exactly.

![image](https://user-images.githubusercontent.com/37159177/112775796-ed95f580-900b-11eb-9c36-ca873c686b9a.png)